driver/pelican/main.go

```diff
@@ -244,6 +255,22 @@ func main() {
 			}
 		}()
 
+		go func() {
+			for {
+				if schedStatus, schedErr := currentPelican.GetSchedule(); schedErr != nil {
+					fmt.Printf("Failed to retrieve Pelican's Schedule: %v\n", schedErr)
+				} else {
+					fmt.Printf("%s Schedule: %+v\n", currentPelican.Name, schedStatus)
+					po, err := bw2.CreateMsgPackPayloadObject(bw2.FromDotForm(SCHED_PO_DF), schedStatus)
+					if err != nil {
+						fmt.Printf("Failed to create Schedule msgpack PO: %v", err)
+					}
+					currentSchedIface.PublishSignal("info", po)
```

Comment:
@gtfierro In the Pelican's status retrieval loop, we write to the `done` channel if an error occurs, which prompts the driver to exit. We chose not to do this in the DR event retrieval loop because we figured a problem with the DR interface should be logged but not prompt an exit.

Do you have a preference on which approach we take here? As it stands, an error during schedule retrieval will be logged but the driver will keep running.

Reply:
I like the "log but keep going" approach, like we did with DR event retrieval